CinC2016Challenge_PCG/examples/nupic2016/main.m

```diff
@@ -0,0 +1,26 @@
+%% PREPROCESS DATA
+clc; clear; close all
+!./preprocess.sh
+wav2vect
+train = csvread('train.csv');
+pcg=train(:,2);
+pcg_full = csvread('signal.csv');
+pcg_full = pcg_full(:,2);
+
+%% TRAIN HTM model
+enc_min = min(pcg_full) % use these to set params of nupic encoder!
+enc_max = max(pcg_full)
+enc_resol = mean(abs(diff(pcg_full))) %TODO improve this
+
+%add header
+!echo "reset,consumption" > tr.csv
+!echo "int,float" >> tr.csv
+!echo "R," >> tr.csv
+!cat train.csv >> tr.csv
+
+% run HTM model
+!python run_opf_experiment.py model/ -c trained 
```

Comment:
NuPIC does not run like this (paths are set incorrectly when called from matlab), will have to do it the other way: shell script calls python/nupic and matlab code